### PR TITLE
abdalla/add language to deriv banner link - Banner Is visible to EU

### PR DESCRIPTION
**Pull request by abdalla-binary**

### CODE REVIEW

File: src/templates/_common/components/deriv_banner.jsx [hunk 1]
```diff
@@ -3,8 +3,8 @@ import PropTypes from 'prop-types';
 
 const DerivBanner = ({ content_name }) => (
     <React.Fragment>
-        <div id='deriv_banner_container' data-show='-eucountry'>
-            <a target='_blank' rel='noopener noreferrer' href={`https://deriv.com/interim/deriv/?utm_source=binary&utm_medium=referral&utm_campaign=deriv-launch&utm_content=${content_name}`}>
+        <div id='deriv_banner_container'>
+            <a target='_blank' rel='noopener noreferrer' href={`${it.deriv_banner_url}interim/deriv/?utm_source=binary&utm_medium=referral&utm_campaign=deriv-launch&utm_content=${content_name}`}>
```
Comment (negarn) [inline src/templates/_common/components/deriv_banner.jsx:7]: missing `/` before interim

File: scripts/render.js [hunk 2]
```diff
@@ -236,6 +236,7 @@ async function compile(page) {
 
     const tasks = languages.map(async lang => {
         const affiliate_language_code = common.getAffiliateSignupLanguage(lang);
+        const deriv_language_code = lang === 'EN' ? '' : `${lang.toLowerCase()}/`;
```
Comment (negarn) [inline scripts/render.js:239]:
```suggestion
        const deriv_language_code = lang === 'EN' ? '' : `${lang.toLowerCase()}`;
```

File: scripts/render.js [hunk 3]
```diff
@@ -250,6 +251,7 @@ async function compile(page) {
             affiliate_signup_url  : `https://login.binary.com/signup.php?lang=${affiliate_language_code}`,
             affiliate_password_url: `https://login.binary.com/password-reset.php?lang=${affiliate_language_code}`,
             affiliate_email       : 'partners@binary.com',
+            deriv_banner_url      : `https://deriv.com/${deriv_language_code}`,
```
Comment (negarn) [inline scripts/render.js:254]:
```suggestion
            deriv_banner_url      : `https://deriv.com${deriv_language_code}`,
```

File: src/templates/_common/components/deriv_banner.jsx (hunk 1 above)
Comment (negarn) [inline src/templates/_common/components/deriv_banner.jsx:7]:
```suggestion
            <a target='_blank' rel='noopener noreferrer' href={`${it.deriv_banner_url}/interim/deriv/?utm_source=binary&utm_medium=referral&utm_campaign=deriv-launch&utm_content=${content_name}`}>
```

**Review by negarn — APPROVED** [review]

**Review by negarn — APPROVED** [review]

File: scripts/render.js [hunk 4]
```diff
@@ -236,6 +236,7 @@ async function compile(page) {
 
     const tasks = languages.map(async lang => {
         const affiliate_language_code = common.getAffiliateSignupLanguage(lang);
+        const deriv_language_code = lang === 'EN' ? '' : `${lang.toLowerCase().replace(/_/g, '-')}/`;
```
Comment (negarn) [inline scripts/render.js:239]: do we still need this or redirect from /en to / is done in deriv.com?

Comment (abdalla-binary) [reply scripts/render.js:239]: deriv.com now handles that, but it overwrites the whole like which overwrites the affiliate details provided with the url

**Review by negarn — APPROVED** [review]

**Review by negarn — APPROVED** [review]

**Pull request merged**